sdk/schemaregistry/schema-registry/rollup.base.config.js

```diff
@@ -0,0 +1,103 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
+import multiEntry from "@rollup/plugin-multi-entry";
+import cjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
+import { terser } from "rollup-plugin-terser";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import viz from "rollup-plugin-visualizer";
+
+const pkg = require("./package.json");
+const depNames = Object.keys(pkg.dependencies);
+const input = "dist-esm/src/index.js";
+const production = process.env.NODE_ENV === "production";
+
+export function nodeConfig(test = false) {
+  const externalNodeBuiltins = ["events"];
```

Comment:
We don't depend on `events`, so we can keep this as an empty array as a placeholder for future external node builtins

```suggestion
  const externalNodeBuiltins = [];
```